UPGRADE.md

```diff
@@ -0,0 +1,7 @@
+This document describes future changes that will infect your current resources used inside of OpenShift.
+Each change contains description of the change and a date when that change will happen.
+
+* Build's pod label will be renamed from `build` to `io.openshift.build.name`.
+  All the newly created Builds will already have the new label, but old ones will
+  be recognized (inside the build controller) until post 3.1/1.2 version.
```

Comment:
Let's say that we plan on removing this in the release that corresponds to
3.1.  The upgrade instructions for 3.1 will contain more info about making
this migration possible (probably by upgrading to the latest 3.0.x)

On Tue, Jun 30, 2015 at 10:09 AM, Ben Parees <notifications@github.com>
wrote:

> In UPGRADE.md
> <https://github.com/openshift/origin/pull/3508#discussion_r33576024>:
>
> > @@ -0,0 +1,7 @@
> > +This document describes future changes that will infect your current resources used inside of OpenShift.
> > +Each change contains description of the change and a date when that change will happen.
> > +
> > +* Build's pod label will be renamed from `build` to `io.openshift.build.name`.
> > +  All the newly created Builds will already have the new label, but old ones will
> > +  be recognized (inside the build controller) until post 3.1/1.2 version.
>
> "old values will be recognized by OpenShift through the 3.1 release.
> Support for these values may be removed in any major point release after
> 3.1 (e.g. 3.2)"
>
> @smarterclayton <https://github.com/smarterclayton> please confirm that's
> the version guarantee we want to make here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3508/files#r33576024>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift